recipe/meta.yaml

```diff
@@ -21,14 +21,14 @@ build:
 
 requirements:
   build:
-    - python
+    - python ==python2.7
```

Comment:
I was just trying it out as a PR only in this branch. It got me no joy, and is not on the master branch. I will delete this PR if it makes things better though. ;-)